config/jail.conf

```diff
@@ -767,3 +767,18 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow]
+# allow FTP traffic after successful HTTP auth
+enabled    = false
+filter     = apache-pass
+banaction  = iptables-multiport
+blocktype  = RETURN
+returntype = DROP
+# access log of the website with HTTP auth
+logpath    = /var/log/apache2/access.log
```

Comment:
I'm still not sure I see the value in restricting this default jail to debian, and even further to only one access log.   If you have apache setup with authentication, any other site should yield a 404 or 401...so it wouldn't grant access.  More complicated setups (site-per-service using the same /knocking path) should know better to configure fail2ban properly.